src/sentry/identity/vsts/provider.py

```diff
@@ -48,7 +58,6 @@ def get_pipeline_views(self):
                 client_id=self.get_oauth_client_id(),
                 client_secret=self.get_oauth_client_secret(),
```

Comment:
I don't think `VSTSOauth2CallbackView` actually sends the user anywhere on completion, does it? I vaguely remember it just whitescreening when I removed the last step.

Reply:
Is that a problem? It seems to just call pipeline.next_step(), which is either finish_pipeline() if it's the last step, or the next view (which is the case for login)

Reply:
I guess not, now that I look at it. I wasn't using this within a popup, so I think the resulting template was just rendering the JS to close the popup (which it wasn't in, so it just rendered nothing).